src/Client/app/(root)/my-exams/start-exam/page.tsx

```
@@ -0,0 +1,262 @@
+                                </div>
+                                <div className="text-right">
+                                    <span className="text-[#71717a] dark:text-white">
+                                        MCQ:
+                                    </span>
+                                    {currentExam?.mcqPoints}
+                                </div>
+                                <div>
+                                    <span className="text-[#71717a] dark:text-white">
+                                        Duration:
+                                    </span>
+                                    {formatTimeHourMinutes(parseInt(currentExam?.durationMinutes))} hr
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)